docs/api-index.hbs

```diff
@@ -0,0 +1,25 @@
+{{#class name="Wechaty"~}}
```

Comment:
Please move all `*.hbs` file to `docs/config/` directory, to give them a more clear location to self-document.

Reply:
And this `hbs` is not official part. So I suggest to seperate `api-index.hbs` with the other two